includes/Renderer/Stories/FieldStateFactory/Factory.php

```diff
@@ -119,8 +119,6 @@ public function get_field( $view = 'grid' ) {
 				 * @param FieldState $default_field_state Field state object.
 				 */
 				$field_state = apply_filters( 'web_stories_default_field_state', $default_field_state );
-
-				$field_state = $field_state instanceof FieldState ? $field_state : $default_field_state;
```

Comment:
Why was this removed? This seems valid to me. If someone filters a field_state to false, we need to reset it back to a class that extends `FieldState`

Reply:
With `ApplyFiltersDynamicFunctionReturnTypeExtension` in place, the filter's PHPDoc becomes the source of truth.

The docblock says this can only ever be a `FieldState` instance, so that's the only thing PHPStan expects and it would yell at us if we'd _still_ do an `instanceof` check.

If someone filters this to be something else, they deserve to get an error.

The alternative is to do something like `@param FieldState|mixed $default_field_state Field state object.` but I don't see a point in doing that.